C/README.md

```diff
@@ -172,6 +172,8 @@ _add list here_
 - [Happy Number](math/happy_number.c)
 - [Automorphic Number](math/automorphic_no.c)
 - [Karpekar Number](math/Kaprekar_no.c)
+- [Neon Number](math/Neon_number.c) 
```

Comment:
Remove this `- [Neon Number](math/Neon_number.c) ` why you adding this as the file already present.